Package/PackageActions/Create.cs

```diff
@@ -111,7 +115,26 @@ private int Process(string[] OutputPaths, CancellationToken cancellationToken)
                         log.Error("Package name cannot contain invalid file path characters: '{0}'.", pkg.Name[illegalCharacter]);
                         return (int)PackageExitCodes.InvalidPackageName;
                     }
-                    
+
+                    HashSet<char> warned = new HashSet<char>();
+                    foreach (var ch in pkg.Name.Where(c => DiscouragedPackageNameChars.Contains(c)))
+                    {
+                        if (warned.Add(ch))
+                        {
+                            switch (ch)
+                            {
+                                case '/':
+                                    log.Warning($"Use of the character '/' in a package name is discouraged. " +
+                                                $"This will never be supported by the package repository, " +
+                                                $"and might lead to problems in the future.");
+                                    break;
+                                default:
+                                    log.Warning($"Use of the character '{ch}' in a package name is discouraged.");
+                                    break;
+                            }
+                        }
+                    }
```

Comment:
Can this be simplified to this?
```suggestion
                    foreach (var ch in pkg.Name.Distinct())
                    {
                            switch (ch)
                            {
                                case '/':
                                    log.Warning($"Use of the character '/' in a package name is discouraged. " +
                                                $"This will never be supported by the package repository, " +
                                                $"and might lead to problems in the future.");
                                    break;
                                default:
                                    break;
                            }
                    }
```